Library/Homebrew/dev-cmd/bump-revision.rb

```diff
@@ -18,9 +18,13 @@ def bump_revision_args
       EOS
       switch "-n", "--dry-run",
              description: "Print what would be done rather than doing it."
+      switch "--write",
+             description: "Make the expected file modifications without taking any Git actions."
```

Comment:
`--write-only` feels more appropriate here; feels weird for `--write` to do less? `--no-git` or something may make sense, too.

Reply:
I was trying to be consistent with `bump-formula-pr` but I agree, `--write-only` makes more sense. I prefer it to `--no-git`.

Reply:
@Moisan Consistency is good! What do you think about adding a comment for deprecating `--write` there and adding it as an alias for now?

Reply:
Good idea, should I do that in another PR?

Reply:
@Moisan Up to you!

Reply:
Looks like `brew type check --update` may need run first?

Reply:
I don't think `brew typecheck --update` is what you want here.

You'll need to add the following to [`cli/args.rbi`](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/cli/args.rbi):

```rb
      sig { returns(T::Boolean) }
      def write_only?; end
```

Reply:
I can possibly add an RBI generator for that at some point.

It should probably be scoped to specific commands rather than what we have now.